### PR TITLE
adding first draft of java guide

### PR DESCRIPTION
**Pull request by ChristinaLK**

### CODE REVIEW

File: examples/java-on-osg.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+If your code uses Java via a `.jar` file, it is easy to bring along your 
+own copy of the Java Development Kit (JDK) which allows you to run 
+your `.jar` file anywhere on the OSG. 
+
+## Steps to Use Java on the OSG
+
+1. **Get a copy of Java/JDK.** You can access the the Java Development Kit (JDK) from 
+the [JDK website](https://jdk.java.net/). First select the link to the 
+JDK that's listed as "Ready for Use" and then download the Linux/x64
```
Comment (rachellombardi) [inline examples/java-on-osg.md:15]:
```suggestion
JDK that is listed as "Ready for Use" and then download the Linux/x64 
```

File: examples/java-on-osg.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+If your code uses Java via a `.jar` file, it is easy to bring along your 
+own copy of the Java Development Kit (JDK) which allows you to run 
+your `.jar` file anywhere on the OSG. 
+
+## Steps to Use Java on the OSG
+
+1. **Get a copy of Java/JDK.** You can access the the Java Development Kit (JDK) from 
+the [JDK website](https://jdk.java.net/). First select the link to the 
+JDK that's listed as "Ready for Use" and then download the Linux/x64 
+version of the tar.gz file. The downloaded file should end up in your
```
Comment (rachellombardi) [inline examples/java-on-osg.md:16]:
```suggestion
version of the tar.gz file using a Unix command such as `wget` from your `/home` directory. 
For example, 

$ wget https://download.java.net/java/GA/jdk17.0.1/2a2082e5a09d4267845be086888add4f/12/GPL/openjdk-17.0.1_linux-x64_bin.tar.gz

The downloaded file should end up in your 
```

Comment (rachellombardi) [reply examples/java-on-osg.md:16]: I'd put line 20 in a code chunk (GitHub edits won't let me do it in a suggestion).

**Review by rachellombardi — COMMENTED** [review]
I think this looks great. Let me know if you want me to look at it again.

File: examples/java-on-osg.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
+[title]: - "Using Java on the OSG"
```
[the remark this answers is not in the document]
Comment (ChristinaLK) [reply examples/java-on-osg.md]: @rachellombardi made changes, what do you think?

**Review by showmic09 — COMMENTED** [review]
LGTM. Don't have any experience with Java though

File: examples/java-on-osg.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+		tar -xzf openjdk-17.0.1_linux-x64_bin.tar.gz
+		# Add the unzipped JDK folder to the environment
+		export PATH=$PWD/jdk-17.0.1/bin:$PATH
+		export JAVA_HOME=$PWD/jdk-17.0.1
+	
+		# run your .jar file
+		java -jar program.jar
+
+	Note that the exact name of the unzipped JDK folder and the JDK tar.gz file will 
+	vary depending on the version you downloaded. You should unzip the JDK tar.gz
```
Comment (rachellombardi) [inline examples/java-on-osg.md:47]:
```suggestion
	**Note that the exact name of the unzipped JDK folder and the JDK tar.gz file will 
	vary depending on the version you downloaded.** You should unzip the JDK tar.gz 
```

**Review by rachellombardi — COMMENTED** [review]
Almost done!

**Pull request merged**